packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputEditor.js

```diff
@@ -90,11 +92,23 @@ export default class InputEditor extends Component {
       'Python'
     ].filter(isString).map(o => ({ label: o, value: o }));
 
+    if (injector) {
+      this.translate = this.translate ? this.translate : injector.get('translate');
```

Comment:
I'd rather see `translate` assignment in the constructor instead of caching within render method.